sw-emulator/lib/periph/src/root_bus.rs

```diff
@@ -331,17 +344,29 @@ impl CaliptraRootBus {
     pub const ICCM_SIZE: usize = 256 * 1024;
     pub const DCCM_SIZE: usize = 256 * 1024;
 
-    pub fn new(clock: &Clock, mut args: CaliptraRootBusArgs<'_>) -> Self {
+    pub const UART_NOTIF_IRQ: u8 = 16;
```

Comment:
I don't think we'll need these in this root bus.

Reply:
This is used at line 367 for the IRQ.

Reply:
Removed.